config.yaml

```diff
@@ -1,7 +1,7 @@
 # Number of worker processes to spawn.
 # Set to 0 to run everything in a single process without clustering.
 # Use 'ncpu' to run as many workers as there are CPU units
-num_workers: ncpu
+num_workers: 0
```

Comment:
This config is not supposed to be used for realz and is in here as an example, why are you making this change

Reply:
Indeed.  This was left over from testing during development.  Reverted.